src/lib/caches/WFInfo.js

```
@@ -0,0 +1,89 @@
+    return undefined;
+  }
+
+  static get prices() {
+    if (settings.wfInfo?.prices) {
+      return this.#cache.getKey('prices');
+    }
+    return undefined;
+  }
+
+  static populate(logger = Logger('WFINFO')) {
+    return this.#hydrate(logger);
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)